index.d.ts

```diff
@@ -0,0 +1,72 @@
+// Type definitions for react-responsive-context
+// Project: react-responsive-context
+// Definitions by: Filipe Medeiros filipesilvamedeiros@gmail.com
```

Comment:
You don't need to include those comments here.
People normally add a comment at the top of the d.ts when adding it to the `DefinitelyTyped` repo

Reply:
Done!